example/eject.js

```diff
@@ -16,20 +17,20 @@
 
 'use strict';
 
-const mountutils = require('.');
-const drive = process.argv[2];
+const mountUtils = require('..');
 
-if (!drive) {
-  console.error(`Usage: ${process.argv[0]} ${process.argv[1]} <drive>`);
+let argv = process.argv.slice(2);
+let disk = argv.shift();
```

Comment:
Doesn't matter, really – they could be `const`, `let` or `var` and it would work just the same.
Fun thing: Even though `const` is supposed to signal an immutable value, the value's values can still change, but reassignment throws:
```js
> const a = [ 1, 2, 3, 4, 5, 6 ]
undefined
> a.shift()
1
> a
[ 2, 3, 4, 5, 6 ]
> a = 'not an array'
TypeError: Assignment to constant variable.
```